src/main/java/org/apache/mesos/curator/CuratorConfigStore.java

```diff
@@ -183,6 +182,10 @@ public UUID getTargetConfig() throws ConfigStoreException {
         }
     }
 
+    public void close() {
```

Comment:
Tried making them package private.  I'll try again.  Tests didn't have access.  You're right though.  Tests were failing because so many curatorpersister's were created that the simultaneous connection limit was hit for ZK.